templates/comments/media-js.html

```diff
@@ -169,9 +169,9 @@
                                 if (window.add_code_copy_buttons)
                                     window.add_code_copy_buttons($area);
                                 var $edits = $comment.find('.comment-edits').first();
-                                $edits.text('updated');
+                                $edits.text('{{ _('updated') }}');
                             }).fail(function () {
-                                console.log('Failed to update comment:' + id);
+                                console.log('Failed to update comment: ' + id);
```

Comment:
Should this receive the same treatment?

Reply:
I left it out because the user never sees `console.log`'s output.

Reply:
I think we should change this to an `alert`.

Reply:
Done. The user will see `Failed to update comment body.`